tools/vf-component-generator/templates/_README.md

```diff
@@ -1,7 +1,7 @@
-👋 Hi component authour, you've read the docs, right?
+👋 Hi component author, you've read the docs, right?
```

Comment:
(not a blocker)

Is there an extension you're using to change to US English? 

As EMBL is pan-eurpoean International English is the adopted version.

Reply:
Here's a link on it https://latest.visual-framework.dev//guidance/language-british-american-english/index.html?q=British

Reply:
Is that true? I tried searching everywhere and its author, even wiktionary says it's an obsolete term for author.
![image](https://user-images.githubusercontent.com/3931042/115023927-055eec00-9edd-11eb-8215-090605ebe90f.png)

while Cambridge dictionary has no such word.
What should I do? 


> Nice. looking good. Are you done writing code?
> 
> if so I'll open this on my local and make some final suggestions with a PR for this PR.

Done, feel free to add code PR.

Reply:
oh my -- I think I need an afternoon coffee -- completely read this incorrectly :D 

What's been edited in your PR is correct :D 

For 'author' it is indeed 'author' not 'authour'.

But colour should be colour etc etc still -- which isn't the issue here :D  

Resolving this conversation and going to drink a double espresso.